examples/psbt_sign_finalize.rs

```diff
@@ -28,10 +29,6 @@ fn main() {
         "Bridge address: {}",
         bridge_descriptor.address(Network::Regtest).unwrap()
     );
-    println!(
```

Comment:
My bad, I thought this would be the weight of the max weight algorithm. So I assumed that this would be changed now since we are planning only with few available assets. But now I read the doccomment and got to know about this being an upper bound. I will get this back in.